Plugins/GitUIPluginInterfaces/ViewModels/RepoModifiedMessage.cs

```diff
@@ -0,0 +1,3 @@
+namespace Plugins.GitUIPluginInterfaces.ViewModels;
+
+public sealed record RepoModifiedMessage();
```

Comment:
```suggestion
public sealed record RepositoryModifiedMessage();
```

What does it mean? 
Please add xml-docs.

Reply:
done